miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/MiniAppSecureStorageDispatcher.kt

```diff
@@ -2,38 +2,34 @@ package com.rakuten.tech.mobile.miniapp.js
 
 import android.app.Activity
 import androidx.annotation.VisibleForTesting
-import androidx.lifecycle.Observer
 import com.google.gson.Gson
 import com.rakuten.tech.mobile.miniapp.errors.MiniAppSecureStorageError
 import com.rakuten.tech.mobile.miniapp.storage.MiniAppSecureStorage
-import com.rakuten.tech.mobile.miniapp.storage.StorageState
+
+internal const val DB_NAME_PREFIX = "rakuten-"
 
 @Suppress("TooManyFunctions", "LargeClass")
 internal class MiniAppSecureStorageDispatcher(
     private val storageMaxSizeKB: Int
 ) {
-    private lateinit var bridgeExecutor: MiniAppBridgeExecutor
-    private lateinit var activity: Activity
+    private val databaseVersion = 1
```

Comment:
I think, this property can be initiated from `MiniAppSecureStorage`.

Reply:
I've assigned The `databaseVersion` in the `MiniAppSecureStorageDispatcher` it's because just in case if in the future the Database versions can be decided via Manifest or something like `storageMaxSizeKB`